### PR TITLE
SSH port

### PR DESCRIPTION
**Pull request by m-zare**
Now it is possible to use non standard ssh port.

### CODE REVIEW

File: library/ssh_copy_id.py [hunk 1]
```diff
@@ -94,7 +98,8 @@ def run_module():
     username = module.params['username']
     password = module.params['password']
     public_key = module.params['ssh_public_key']
-
```
Comment (ryankwilliams) [inline library/ssh_copy_id.py:97]: Could this carriage return be removed? This way then all the local variables are combined together.

**Review by ryankwilliams — CHANGES_REQUESTED** [review]

**Comment by ryankwilliams** [comment]
Hi @m-zare! Thank you for submitting this PR. Everything looks good to me code wise. I suggested one small styling fix in the python module. The other thing is could this new role variable _port_ be documented in the README.md under the _Role Variables_ section? This way then other consumers of the role can know about this.

Thanks!

i.e.

| Variable | Description |
| --- | --- |
| port | SSH port to connect to |

File: library/ssh_copy_id.py (hunk 1 above)
Comment (m-zare) [reply library/ssh_copy_id.py:97]: It's done.

**Comment by m-zare** [comment]
All done. 
@rywillia

**Review by ryankwilliams — APPROVED** [review]
LGTM

**Pull request merged**